doc/custom-configuration.md

```diff
@@ -91,3 +91,38 @@ spec:
 
 Cassandra will load the `cassandra.yaml.d/100-concurrent.yaml` file as well as the default settings managed by the operator!
 
+### Prometheus support
+
+The Cassandra image that is included with this project has the [cassandra-exporter](https://github.com/instaclustr/cassandra-exporter) built in, so it is ready to export the metrics to Prometheus instance. In order to use this capability automatically, one must have a running Prometheus operator in the same Kubernetes environment and have Prometheus object following a set of ServiceMonitors via a selector (a set of labels); for example:
```

Comment:
Yea, I can agree that we don't need to explain how to deploy Prometheus. The reason I did it here, is because for me, as I never actually ran Prometheus before, it was somewhat complicated to understand the flow of things. So I'll remove it for now, but if people ask for clarification, I reckon we can either put it back or at least reference the official Prometheus docs.

Also, the fact that labels put on _some_ service are _also_ put on the `ServiceMonitor` is purely circumstantial, and users definitely shouldn't have to be aware of this detail of the implementation. The CRD parameter is called `prometheusServiceMonitorLabels` for a reason - it's the set of labels used in their own selector in the Prometheus object (which they still can configure any way they want), and that's how they should think about it. It's the set of labels that is going to be used to label the ServiceMonitor we create so that Prometheus picks it up.